src/Internal/Payment/PaymentRequest.php

```diff
@@ -89,4 +106,42 @@ public function get_payment_method_id(): ?string {
 			? sanitize_text_field( wp_unslash( ( $this->request['payment_method_id'] ) ) )
 			: null;
 	}
+
+	/**
+	 * Gets payment method object from request.
+	 *
+	 * @throws PaymentRequestException
+	 */
+	public function get_payment_method(): PaymentMethodInterface {
+		$request = $this->request;
+
+		$is_woopayment_selected = isset( $request['payment_method'] ) && WC_Payment_Gateway_WCPay::GATEWAY_ID === $request['payment_method'];
```

Comment:
Wrong operator here, should be `AND`.

Reply:
I think I had fixed it in this commit 00eb0c0674837a5cea7a83370bab9a04ad2e3a47 before you commented here.

Currently, it's `&&` operator.

Reply:
Oh, it was shortly after I `git pull`-ed last. Nice!